models/intermediate/int_zendesk__schedule_spine.sql

```diff
@@ -105,11 +105,11 @@ with timezone as (
 -- Now we need take holiday's into consideration and perform the following transformations to account for Holidays in existing schedules
 ), holiday_start_end_times as (
 
-    select 
+    select
         calculate_schedules.*,
         schedule_holiday.holiday_name,
-        schedule_holiday.holiday_start_date_at,
-        cast({{ dbt.dateadd("second", "86400", "schedule_holiday.holiday_end_date_at") }} as {{ dbt.type_timestamp() }}) as holiday_end_date_at, -- add 24*60*60 seconds
+        cast({{ dbt.dateadd("minute", "offset_minutes_to_add", "schedule_holiday.holiday_start_date_at") }} as {{ dbt.type_timestamp() }}) as holiday_start_date_at,
+        cast({{ dbt.dateadd("minute", "1440 - offset_minutes_to_add", "schedule_holiday.holiday_end_date_at") }} as {{ dbt.type_timestamp() }}) as holiday_end_date_at, -- add 24*60 = 1440 minutes
```

Comment:
Could you explain the change you're making here? Im curious about the `second` to `minute` switch as well as the `1440 - offset_minutes_to_add`

Reply:
I changed from `second` to `minute` to account for the fact that the offset is stored in minutes (the alternative could be to keep it in seconds, 86400 and then multiple offset_minutes by 60).  Figured since it was a hard coded time to add, hardcoding it to 1440 and the not having to multiply the offset minutes made more sense.

`1440 - offset_minutes_to_add` is mimicking the logic applied to the start/time, which subtracts the offset so we can compare the two set of times:
 

<img width="744" alt="image" src="https://github.com/fivetran/dbt_zendesk/assets/89489939/d725a853-8c58-481e-86d1-a97a7e2b95bd">


potentially adding the `_to_add` to the `offset_minutes` and then subtracting it in that statement is a bit confusing,  especially since our value is -240, currently.  Most definitely open to a rename here.

Reply:
Awesome, thanks for explaining!